packages/@coorpacademy-components/src/molecule/go1-base-banner/index.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ButtonLink from '../../atom/button-link';
+import style from './style.css';
+
+const Go1BaseBanner = props => {
```

Comment:
Not sure about the name.
Would rather use Coorpmanager as a prefix: `CoorpmanagerBaseBanner`